package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "build": "next build",
+    "build": "yarn prebuild && next build",
```

Comment:
I think this is what I was missing in: #2123

https://yarnpkg.com/migration/guide#explicitly-call-the-pre-and-post-scripts